README.md

```diff
@@ -36,24 +36,24 @@ Seneca()
   // Get API keys using the seneca-env plugin
   .use('env', {
     var: {
-      $TANGOCARD_APIKEY: String,
-      $TANGOCARD_USERTOKEN: String,
+      $ORBIT_API_TOKEN: String,
+      $WORKSPACE: String,
     }
   })
   .use('provider', {
     provider: {
       tangocard: {
         keys: {
-          apikey: { value: '$TANGOCARD_APIKEY' },
-          usertoken: { value: '$TANGOCARD_USERTOKEN' },
+          key: { value: '$ORBIT_API_TOKEN' },
+          workspace: { value: '$WORKSPACE' },
         }
       }
     }
   })
-  .use('tangocard-provider')
+  .use('orbit-provider')
 
-let board = await seneca.entity('provider/tangocard/board')
-  .load$('<tangocard-board-id>')
+let board = await seneca.entity('provider/orbit/board')
```

Comment:
change this to an orbit specific example

Reply:
ok, swapped